### DISCUSSION TITLE
Otopark Yeri Çözümü

### DISCUSSION THREAD

**Discussion by yukseel** [sonsuzus/sonsuzus#410] [Zeka Sorusu]
Bir otoparkta 20 boş park yeri var ve 5 farklı otomobil bu park yerlerinden birini seçmek istiyor. Her otomobil aynı anda bir park yeri seçebilir mi yoksa sırayla mi seçmelidirler? Eğer sırayla seçmelilerse, kaç farklı sıralama mümkün olur?
(2 araç aynı yere park edemez :D)

**Comment by akgembi12** [comment]
Toplamda her birinin herhangi nir yere park edebileceğini düşünürsek toplam 20x19x18x17x16 adet ihtimal olur.
 aynı anda seçmekten ne kastettiğini anlayamadım.  :)
